one_click_kag_server/main.py

```diff
@@ -204,6 +204,11 @@ def setup_kag(config: dict, state: State):
     sftp.mkdir("Mods", ignore_existing=True)
     sftp.put_dir("Mods", "Mods")
 
+    # Upload cache
+    logging.info("Uploading cache...")
```

Comment:
I think we need an extra line here to create a Cache directory locally if it doesn't exist, else you get 

```
FileNotFoundError: [Errno 2] No such file or directory: 'Cache'
```

Alternatively add to the instructions that you need to make a `Cache` dir locally before you start. But since it's an edge use case I think it's better just to make one here automatically.

Reply:
Doesn't `Mods` require the same treatment then? You're trying to find the Mods dir locally on line 205 regardless of if there are mods in the config.